pxr/usd/bin/usddiff/usddiff.py

```diff
@@ -170,16 +172,33 @@ def listFiles(dirpath):
                     for file in files]
         return set(ret)
 
-    # Must have FILE FILE, DIR DIR, DIR FILES... or FILES... DIR.
+    origArgs = args
+
+    # path may not be a "real" path due to custom resolvers... so use
+    # Sdf.Layer.FindOrOpen first
```

Comment:
I'm concerned about the performance impact of using Sdf.Layer.FindOrOpen to try to deal with custom asset paths that may not have a corresponding file on the filesystem. It seems like we don't really care about loading the layer here, we just want to know whether a layer at that path actually exists. So instead of doing this, could we try to use Ar.Resolver.Resolve somehow to give us this information?